### PR TITLE
Add 2 header

### PR DESCRIPTION
**Pull request by genakorytov**
Can you suggest what needs to be fixed?

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://genakorytov.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://genakorytov.github.io/layout_moyo-header/report/html_report/)
```
Comment (OlEzhA0) [inline readme.md:4]: copy this links to your PR description

File: src/index.html [hunk 2]
```diff
@@ -3,12 +3,98 @@
...
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="first">
+        <img
+        class="icon" 
+        src="images/logo.png" 
+        alt="Moyo"
+        >
```
Comment (OlEzhA0) [inline src/index.html:18]:
```suggestion
        <img
          class="icon" 
          src="images/logo.png" 
          alt="Moyo"
        >
```

**Review by OlEzhA0 — CHANGES_REQUESTED** [review]
we need only 1 header. that stretched for full width. and don't change background, just leave default white
Screenshot 2023-03-03 at 11 12 28 AM

**Comment by genakorytov** [comment]
- [DEMO LINK](https://genakorytov.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://genakorytov.github.io/layout_moyo-header/report/html_report/)

File: src/index.html [hunk 3]
```diff
@@ -3,12 +3,53 @@
...
+          <li>
+            <a href="http://VIDEO.html">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+      <img
+          class="icon" 
+          src="images/logo.png" 
+          alt="Moyo"
+        >
```
Comment (VitaliyBondarenko1982) [inline src/index.html:52]: logo should be before nav

File: src/index.html [hunk 4]
```diff
@@ -3,12 +3,53 @@
...
-    <h1>Moyo header</h1>
+    <header class="searchBar">
+      <nav class="first">
+        <ul class="link">
+          <li>
+            <a class="is-active" href="http://APPLE.html">APPLE</a>
+          </li>
+
+          <li>
+            <a href="http://SAMSUNG.html">SAMSUNG</a>
```
Comment (VitaliyBondarenko1982) [inline src/index.html:20]:
```suggestion
            <a href="#samsung">Samsung</a>
```
please fix all links

File: src/style.css [hunk 5]
```diff
@@ -1 +1,105 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(Roboto/Roboto-Black.ttf);
```
Comment (VitaliyBondarenko1982) [inline src/style.css:3]: where did the Roboto-Black.ttf file come from?
wrong path to file

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
please first of all compare design and your solution.
also compare in test reference and test example.
if you will have any questions - feel free to ask in fe_chat3

Screenshot from 2023-03-06 21-45-49

**Pull request closed without merging**